### PR TITLE
Feature/rad 131 migrate gradle docker to use GitHub actions instead of travis ci

### PR DESCRIPTION
**Pull request by lewis-russell**

### CODE REVIEW

File: .github/workflows/gradle-docker-ci.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+    - name: Run Unit Tests
+      run: ./gradlew build
+    - name: Publish
+      if: github.ref == 'refs/heads/master'
```
Comment (thomas-anderson-bsl) [inline .github/workflows/gradle-docker-ci.yml:29]: I'd kind of prefer if we did test and publish in separate workflows so it's more clear what's happening.

File: .github/workflows/gradle-docker-ci.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+##
```
Comment (thomas-anderson-bsl) [inline .github/workflows/gradle-docker-ci.yml:1]: Could you please bump the `com.gradle.plugin-publish` version in `build.gradle`, and add the `maven-publish` plugin too please

We essentially want to mirror the changes that Ben has done in here: https://github.com/brightsparklabs/gradle-docs/pull/7/files

File: .github/workflows/gradle-docker-ci.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        java-version: '11'
+        distribution: 'temurin'
+    - name: Run Unit Tests
+      run: ./gradlew build
+    - name: Publish
+      if: github.ref == 'refs/heads/master'
+      run: |
+        mkdir -p ~/.gradle
+        echo "gradle.publish.key=${GRADLE_PUBLISH_KEY}" >> ~/.gradle/gradle.properties
+        echo "gradle.publish.secret=${GRADLE_PUBLISH_SECRET}" >> ~/.gradle/gradle.properties
```
Comment (thomas-anderson-bsl) [inline .github/workflows/gradle-docker-ci.yml:33]: This way of referencing the GRADLE_PUBLISH_X variables won't work. See how to do it here: https://github.com/brightsparklabs/gradle-docs/pull/7/files

**Review by thomas-anderson-bsl — CHANGES_REQUESTED** [review]
Few minor changes.
Essentially replicate the PR here please: https://github.com/brightsparklabs/gradle-docs/pull/7/files

**Comment by al-jeyapal** [comment]
This is closed as the repo has been migrated to use the new actions already.

**Pull request closed without merging**